smartling-api-commons/src/main/java/com/smartling/api/v2/client/CloseClientInvocationHandler.java

```diff
@@ -0,0 +1,35 @@
+package com.smartling.api.v2.client;
+
+import lombok.extern.slf4j.Slf4j;
+
+import javax.ws.rs.client.Client;
+import java.lang.reflect.InvocationHandler;
+import java.lang.reflect.Method;
+
+/**
+ *
+ */
+@Slf4j
+public class CloseClientInvocationHandler  implements InvocationHandler
```

Comment:
2 spaces before implement;
empty comment is redundant;
Slf4j logger defined but not used

Reply:
Fixed.